src/components/fields/SchemaField.js

```diff
@@ -105,7 +115,10 @@ if (process.env.NODE_ENV !== "production") {
   SchemaField.propTypes = {
     schema: PropTypes.object.isRequired,
     uiSchema: PropTypes.object,
-    registry: PropTypes.object,
+    registry: PropTypes.shape({
+      widgets: PropTypes.object.isRequired,
+      fields: PropTypes.object.isRequired,
```

Comment:
No because it's not predictable (or at least PropTypes don't expose any API for ensuring a map is composed from a given typed data structure).